docs/python/quickstart.md

```diff
@@ -5,4 +5,99 @@ sidenav: side-nav-python.html
 type: markdown
 ---
 
-## Abseil Quickstart (Python)
\ No newline at end of file
+# Abseil Quickstart (Python)
+
+This quick tutorial will get you up and running with Python Abseil. Python
+Abseil works with Python 2 and 3. Please see the
+[Programming Guides](/docs/python/guides) for details about specific modules.
+
+## Installation
+
+To install the package, simply run:
+
+```sh
+pip install absl-py
+```
+
+Or you can install from source via the instructions on
+[github](https://github.com/abseil/abseil-py).
+
+## Creating and Running a Script
+
+Here's an example `hello.py` script that takes a user name and optionally the
```

Comment:
and optionally the number of times to print it as input -> and an optional integer specifying the number of times to print input